src/styles/yas/nest.jl

```diff
@@ -1,6 +1,24 @@
 function n_call!(ys::YASStyle, fst::FST, s::State)
     style = getstyle(ys)
 
+    # With `variable_call_indent`, check if the caller is in the list
+    if caller_in_list(fst, s.opts.variable_call_indent) && length(fst.nodes) > 5
```

Comment:
```suggestion
    if caller_in_list(fst, s.opts.variable_call_indent) && length(fst.nodes::AbstractArray) > 5
```
This solves the last remaining warning.